### PR TITLE
Fix actions dev pdn

### PR DESCRIPTION
**Pull request by fagomezra**
Fix actions dev pdn

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: .github/workflows/push-dev-environment.yaml [hunk 1]
```diff
@@ -51,36 +54,40 @@ jobs:
...
+          IMAGE_NAME_WEBAPP: eden-spending-webapp-dev
+          IMAGE_NAME_HAPI: eden-spending-hapi-dev
+          IMAGE_NAME_HASURA: eden-spending-hasura-dev
+          IMAGE_NAME_WALLET: eden-spending-wallet-dev
           # Postgres
-          POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
-          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
-          POSTGRES_DB: ${{ secrets.POSTGRES_DB }}
-          POSTGRES_DATA: ${{ secrets.POSTGRES_DATA }}
+          POSTGRES_USER: ${{ secrets.DEV_POSTGRES_USER }}
```
Comment (xavier506) [inline .github/workflows/push-dev-environment.yaml:64]: we can use github environments to set different variables for each environment without having to change the env var names

File: utils/meta.mk [hunk 2]
```diff
@@ -3,15 +3,10 @@
 VERSION ?= $(shell git rev-parse --short HEAD)
 CURRENT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 
-IMAGE_NAME_WEBAPP=eden-spending-webapp
-IMAGE_NAME_HAPI=eden-spending-hapi
-IMAGE_NAME_HASURA=eden-spending-hasura
-IMAGE_NAME_WALLET=eden-spending-wallet
-
 DOCKER_REGISTRY=eoscostarica506
```
Comment (xavier506) [inline utils/meta.mk:6]: this value needs to be updated

File: utils/meta.mk [hunk 3]
```diff
@@ -3,15 +3,10 @@
 VERSION ?= $(shell git rev-parse --short HEAD)
 CURRENT_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 
-IMAGE_NAME_WEBAPP=eden-spending-webapp
```
Comment (xavier506) [inline utils/meta.mk:6]: where are the image names being sourced?

Comment (fagomezra) [reply utils/meta.mk:6]: I send in the job to build the image and the job for buil the kubernetes file, this for has a image for dev and another for prod.

File: .github/workflows/push-dev-environment.yaml (hunk 1 above)
Comment (fagomezra) [reply .github/workflows/push-dev-environment.yaml]: you right, i changed the variables and updated the values for de development environment secrets.

File: utils/meta.mk (hunk 2 above)
Comment (fagomezra) [reply utils/meta.mk]: updated

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**